<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SyringeSetting</class>
 <widget class="QDialog" name="SyringeSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="injectionSizeValue">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="injectionSize">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="injectionSizeLeft">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="injectionSizeRight">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="doseTimeLeft">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>210</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="doseTimeValue">
   <property name="geometry">
    <rect>
     <x>397</x>
     <y>210</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="doseTimeRight">
   <property name="geometry">
    <rect>
     <x>547</x>
     <y>210</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="doseTime">
   <property name="geometry">
    <rect>
     <x>407</x>
     <y>150</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cureModelLeft">
   <property name="geometry">
    <rect>
     <x>634</x>
     <y>210</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cureModelValue">
   <property name="geometry">
    <rect>
     <x>704</x>
     <y>210</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cureModelRight">
   <property name="geometry">
    <rect>
     <x>854</x>
     <y>210</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cureModel">
   <property name="geometry">
    <rect>
     <x>714</x>
     <y>150</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>417</x>
     <y>340</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
